jans-fido2/model/src/main/java/io/jans/fido2/ctap/CoseEdDSAAlgorithm.java

```diff
@@ -0,0 +1,31 @@
+package io.jans.fido2.ctap;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum CoseEdDSAAlgorithm {
```

Comment:
EDDSA can be dropped. FIDO never supported, and never will. If EDDSA is present then return error.